controllers/idler/idler_controller.go

```diff
@@ -2,8 +2,11 @@ package idler
 
 import (
 	"context"
+	"fmt"
 	"time"
 
+	"github.com/codeready-toolchain/toolchain-common/pkg/cluster"
+
```

Comment:
no need for a blank line

Reply:
done in [777b019](https://github.com/codeready-toolchain/member-operator/pull/357/commits/777b019b9dc2a2da36de0b0e2754b8012b10f7d8) and sorted imports.